### PR TITLE
Don't send the value of the visibility property and make it deprecated

### PR DESCRIPTION
**Pull request by jiikko**
jmatsu wrote down this contents instead:

It's no longer required to send the `visibility` parameter to the server but we should allow users to set/get the visibility property to avoid breaking changes in API. This PR marked them as deprecated and printed warnings if called.

ss 2022-05-10 at 17 02 58

### CODE REVIEW

File: src/main/groovy/com/deploygate/gradle/plugins/dsl/syntax/DeploymentSyntax.groovy [hunk 1]
```diff
@@ -8,8 +8,6 @@ interface DeploymentSyntax {
 
     void setMessage(@Nullable String message)
 
-    void setVisibility(@Nullable String visibility)
```
Comment (jmatsu) [inline src/main/groovy/com/deploygate/gradle/plugins/dsl/syntax/DeploymentSyntax.groovy:11]: We should never allow breaking changes to API without any deprecation phase. Please make this deprecated and print warnings.

**Comment by jmatsu** [comment]
https://github.com/DeployGate/gradle-deploygate-plugin/issues/138

**Comment by jmatsu** [comment]
I'll take over this task.

**Review by tnj — APPROVED** [review]
LGTM!

**Comment by jmatsu** [comment]
Thanks.

**Pull request merged**